ui-components/autocompleter.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: Autocompleter
+title_nav: Autocompleter
+description: Add a custom autocompleter to TinyMCE 5.0.
+keywords: autcomplete
+---
+
+## Overview and use case
+
+An `autocompleter` enables users to quickly find and select from a pre-populated list of values as they type, leveraging searching and filtering. It provides suggestions to insert while the user is typing into the content. For example, typing **:amp** should show the ampersand item in the menu. Pressing `esc` should close the autocomplete menu.
```

Comment:
Add an example from the emoji plugin?

Reply:
Technically charmap, but yeah that'd be a good idea to add that :+1:

Reply:
Done :smile: